src/zenml/cli/stack_components.py

```diff
@@ -38,13 +38,16 @@ def _component_display_name(
 def _get_stack_component(
     component_type: StackComponentType,
     component_name: Optional[str] = None,
+    active_stack_name: Optional[str] = None,
```

Comment:
Little bit confused by this variable name. it's just which stack to get the component_type from?
I would suggest renaming to `stack_name` and furthermore documenting + enforcing that only one of `{component_name, stack_name}` may be not None, because right now the behavior is unclear, obviously one of them has to take preference, but it's better to make this explicit.

Reply:
Same here, I removed it completely as part of the simplification of CLI commands to avoid the global/local complications.